core/src/main/java/org/apache/carbondata/core/scan/executor/util/QueryUtil.java

```diff
@@ -870,4 +873,23 @@ private static void getChildDimensionOrdinal(CarbonDimension queryDimensions,
       }
     }
   }
+
+  /**
```

Comment:
Move to right location, not in query util

Reply:
Will be refactored in CARBONDATA-2713